packages/mask/dashboard/pages/SetupPersona/PermissionOnboarding/index.tsx

```diff
@@ -0,0 +1,152 @@
+import { makeStyles } from '@masknet/theme'
+import { Icons } from '@masknet/icons'
+import { Trend } from '../../../assets/index.js'
+import { EnhanceableSite, userGuideStatus } from '@masknet/shared-base'
+
+import { delay } from '@masknet/kit'
+import { OnboardingWriter } from '../../../components/OnboardingWriter/index.js'
+import { compact } from 'lodash-es'
+import { TwitterAdaptor } from '../../../../shared/site-adaptors/implementations/twitter.com.js'
+import { requestPermissionFromExtensionPage } from '../../../../shared-ui/index.js'
+
+const useStyles = makeStyles()((theme) => ({
+    card: {
+        position: 'fixed',
+        top: 24,
+        right: 24,
+        padding: theme.spacing(2),
+        border: `1px solid ${theme.palette.maskColor.line}`,
+        borderRadius: 12,
+        maxWidth: 360,
+    },
+    pin: {
+        fontSize: 16,
+        lineHeight: '20px',
+        color: theme.palette.maskColor.main,
+    },
+    skeleton: {
+        background: 'linear-gradient(270deg, #F6F6F6 0%, rgba(217, 217, 217, 0) 94.74%)',
+        width: 190,
+        height: 36,
+        borderRadius: 99,
+        marginLeft: 42,
+    },
+    plugins: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        width: 36,
+        height: 36,
+        borderRadius: 99,
+        // hard color
+        background: '#F0F0F4',
+        marginLeft: 18,
+        marginRight: 18,
+    },
+    more: {
+        transform: 'rotate(90deg)',
+    },
+    pinCard: {
+        marginTop: 18,
+        borderRadius: 8,
+        border: `1px solid ${theme.palette.maskColor.line}`,
+        background: theme.palette.maskColor.bottom,
+        padding: 16,
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+    },
+    trend: {
+        position: 'fixed',
+        top: 206,
+        right: 408,
+    },
+    twitter: {
+        color: theme.palette.maskColor.bottom,
+    },
+
+    typed: {
+        fontSize: 36,
+        lineHeight: 1.2,
+        fontWeight: 700,
+        '& > strong': {
+            color: theme.palette.maskColor.main,
+        },
+    },
+    endTyping: {
+        opacity: 1,
+    },
+}))
+
+export const Component = memo(function Onboarding() {
+    const t = useDashboardTrans()
+    const { classes } = useStyles()
+
+    const theme = useTheme()
+
+    const onSetupTwitter = useCallback(async () => {
+        if (!(await requestPermissionFromExtensionPage(EnhanceableSite.Twitter))) return
+        if (!userGuideStatus[EnhanceableSite.Twitter].value) userGuideStatus[EnhanceableSite.Twitter].value = '1'
+        await delay(300)
+        await browser.tabs.create({
+            active: true,
+            url: TwitterAdaptor.homepage,
+        })
+        window.close()
+    }, [])
+
+    const words = useMemo(() => {
+        return compact([
+            <Typography key="identity">
+                {''}
```

Comment:
```suggestion
```

Reply:
No, Because the typewriter components has const [text, strongText] = fragment.props.children as string[]. I need the strong text.